WordPress/Classes/Services/AccountService.m

```diff
@@ -127,6 +127,14 @@ - (void)removeDefaultWordPressComAccount
     [[NSNotificationCenter defaultCenter] postNotificationName:WPAccountDefaultWordPressComAccountChangedNotification object:nil];
 }
 
+- (BOOL)isDefaultWordPressComAccount:(WPAccount *)account {
+    NSString *uuid = [[NSUserDefaults standardUserDefaults] stringForKey:DefaultDotcomAccountUUIDDefaultsKey];
+    if (uuid.length == 0) {
+        return false;
+    }
+    return [account.uuid isEqualToString:uuid];
```

Comment:
Yeah. Clearly. As you said, Swift *REALLY* make it hard for us to read ObjC code!!